src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -155,7 +155,7 @@ def get_vm_size_completion_list(prefix, action, parsed_args, **kwargs):  # pylin
 
 for scope in ['vm', 'vmss']:
     register_cli_argument(scope, 'no_auto_upgrade', action='store_true', help='by doing this, extension system will not pick the highest minor version for the specified version number, and will not auto update to the latest build/revision number on any scale set updates in future.')
-    register_cli_argument('{} create'.format(scope), 'generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing', arg_group='Authentication')
+    register_cli_argument('{} create'.format(scope), 'generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing. Generally will be stored in the ~/.ssh directory', arg_group='Authentication')
```

Comment:
I would agree with Travis.
According to https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py#L685, it's always generated under that directory.
Maybe 'The keys are generated under the ~/.ssh directory.' instead?